### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by yloson**

### CODE REVIEW

File: src/test/java/api/SalesforceApi.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    public HttpResponse generateLead() throws Exception {
+
+        HttpClient httpclient = new DefaultHttpClient();
+        HttpPost post = new HttpPost("https://test.salesforce.com/services/oauth2/token");
+        List<NameValuePair> params = new ArrayList<NameValuePair>();
+        params.add(new BasicNameValuePair("client_id", "3MVG9AzPSkglhtpvenc1ubvIHNqdSv2B8w9bFU2hNfvAOfxt5ri2SCQo_M.lflR7dukJ5s9LtXu.eZ3KzbmRO"));
+        params.add(new BasicNameValuePair("client_secret", "8952510748675479529"));
+        params.add(new BasicNameValuePair("grant_type", "password"));
+        params.add(new BasicNameValuePair("username", "joinus@wework.com.staging"));
+        params.add(new BasicNameValuePair("password", "A8m*Bz?e3p$H"));
```
Comment (ronits-wework) [inline src/test/java/api/SalesforceApi.java]: I think passwords should be extracted to environment variables for security reasons

File: src/test/java/pageObjects/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    public void login(String username, String password) {
+        getDriver().findElement(usernameLoc).sendKeys(username);
+        getDriver().findElement(passwordLoc).sendKeys(password);
+        getDriver().findElement(login).click();
+        waitForElementVisiblity(titleAfterLogin, 20);
+    }
+
+    public void loginNMDProfile() {
+        String decodedPwd = StringDecrypt.decryptXOR("MVkeWTUVTQFDEVc7", "password");
```
Comment (ronits-wework) [inline src/test/java/pageObjects/LoginPage.java]: should be extracted to an env var

File: src/test/java/api/SalesforceApi.java (hunk 1 above)
Comment (yloson) [reply src/test/java/api/SalesforceApi.java]: you are correct, i have encrypted login password as you can see, however once test will run in circleci i will add them as params and read as properties, thanks!

File: src/test/java/api/SalesforceApi.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                "\"company_size\": \"10\"," +
+                "\"locale\": \"en-US\"," +
+                "\"country_code\": \"IL\"," +
+                "\"desks_interested_min\": 12," +
+                "\"desks_interested_max\": 15," +
+                "\"desks_interested_range\": \"20\"," +
+                "\"lead_source\":" + "\"" + this.leadSource + "\"" +  "," +            // \"WeWork.com\"," +
+                "\"lead_source_detail\": \"Book A Tour Availability\"," +
+                "\"product_line\": \"WeWork\"," +
+                "\"move_in_time_frame\": \"2019-05-01\"," +
```
Comment (yloson) [inline src/test/java/api/SalesforceApi.java]: need to add getDate() function to replace hardcoded date

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+jobs:
+  build:
+    branches:
+      only:
+        - master
+        - salesforce_login
+
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8-jdk
```
Comment (chrisrouland) [inline .circleci/config.yml]: try changing this to selenium/standalone-chrome:3.141.59

File: src/test/java/api/SalesforceApi.java (hunk 1 above)
Comment (yloson) [reply src/test/java/api/SalesforceApi.java]: resolved

File: src/test/java/pageObjects/LoginPage.java (hunk 2 above)
Comment (yloson) [reply src/test/java/pageObjects/LoginPage.java]: replaced with env variable

File: src/test/java/api/SalesforceApi.java (hunk 3 above)
Comment (yloson) [reply src/test/java/api/SalesforceApi.java]: will fix in future pr

File: .circleci/config.yml (hunk 4 above)
Comment (yloson) [reply .circleci/config.yml]: was replaced with the image you found: circleci/openjdk:8-jdk-browsers

**Comment by chrisrouland** [comment]
@yloson can you add a description capturing what the scope of this PR is? What part of the test flow it is supposed to cover, etc.

**Comment by yloson** [comment]
closing old pr

**Pull request closed without merging**